src/components/RadioButton/RadioButton.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import Text from '../Text';
+import styles from './RadioButton.scss';
+import { CHECKBOX_VIEWBOX } from '../../constants';
+
+const { block, elem } = bem({
+    name: 'RadioButton',
+    classnames: styles,
+    propsToMods: ['disabled']
+});
+
+const RadioButton = props => {
+    const { id, children, viewbox, disabled, ...rest } = props;
+
+    return (
+        <div {...block(props)}>
+            <input type="radio" id={id} {...rest} {...elem('input', props)} disabled={disabled} />
```

Comment:
@eszthoff Now that I think of it, we may want `{...rest}` to come before the `type` attribute, otherwise you could for instance change input type to `text` which doesn't make sense for this component. Same for Checkbox.js, actually..

Reply:
moved it